slack-conversation.c

```diff
@@ -35,10 +36,111 @@ static void conversations_list_cb(SlackAccount *sa, gpointer data, json_value *j
 		slack_login_step(sa);
 }
 
+static void get_latest_history_cb(SlackAccount *sa, gpointer data, json_value *json, const char *error) {
+	json_value *chan_json = json_get_prop(json, "channel");
+	if (!chan_json)
```

Comment:
Need to ensure chan_json is the right type: use `json_get_prop_type(json, "channel", object)`